src/domain/pricing/details/sections/prices-details/edit-prices-overrides/index.tsx

```diff
@@ -112,4 +123,13 @@ const EditPricesOverridesModal = ({
   )
 }
 
+const mergeExistingWithDefault = (variantPrices: any[] = [], defaultPrices) => {
+  return defaultPrices.map((pr) => {
+    const price = variantPrices.find(
+      (vpr) => vpr?.currency_code === pr.currency_code
+    )
+    return price || pr
+  })
+}
```

Comment:
comment: This method is defined in two files. Should we move to a util?

Reply:
nice catch! yes totally!